tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -880,7 +880,7 @@ platform_tests/broadcom:
   skip:
     reason: "Marvell devices does not support platform_tests/broadcom"
     conditions:
-      - "'marvell' in asic_type"
+      - "'marvell' in asic_type or 'marvell-prestera' in asic_type"
```

Comment:
"asic_type in ['marvell', 'marvell-prestera']" is the deterministic way?

the check above will include marvel-*

Reply:
Hi @yxieca 

Marvell ASIC name is changed from “marvell” to “marvell-Prestera” however we also have “marvell-teralynx”. If we use “marvell-*”, test cases will fail for marvell teralynx platforms

Thanks
Rajneesh